src/fineract.yaml

```diff
@@ -0,0 +1,30 @@
+---
+description: >
+   Mature platform with open APIs that provides a reliable, robust, and
+   affordable core banking solution for financial institutions offering
+   services to the world’s 2 billion underbanked and unbanked.
+license: ASF 2.0
+name: Fineract
+programmingLanguages:
+  - java
+rating: 7
+repoUrl: "https://github.com/apache/fineract"
+sdgs: [1]
+websiteUrl: "http://fineract.apache.org/"
+
+contributionGuidelinesUrl: "https://cwiki.apache.org/confluence/display/\
+                            FINERACT/Contributor%27s+Zone"
+logoUrl: "https://cwiki.apache.org/confluence/download/attachments/61334077/\
+          FINERACT?version=1&modificationDate=1451316863000&api=v2"
+starsUrl: "https://img.shields.io/github/stars/org/project.svg\
+           ?style=social&label=Star&maxAge=2592000"
```

Comment:
`org/project` should be replaced with `apache/fineract`

Reply:
sorry about that, thanks for the catch!